integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -117,6 +117,20 @@
                         }
                     },
                     {
+                        bidder: 'sonobi',       //  New format
+                        params: {
+                            dom_id: 'PER SLOT', //  <String> dom Id

```

Comment:
DOM stands for "document object model" - the id is the corresponding ID of the ad slot you are targeting.

The id can literally be anything, so long as it is unique to the ad slot.  Some publishers make up their own ID's for their own purposes, others use the div ID's provided by DFP.

The DFP id's often follow the pattern "div-gpt-ad-123456789-1", as you might see here:
`<div id="div-gpt-ad-123456789-1">
<script type="text/javascript">
googletag.cmd.push(function() { googletag.display("div-gpt-ad-123456789-1"); });
</script>
</div>`

Sonobi uses the ID provided by you to differentiate specific ad slots on your page, as we have noted that many publishers choose to use the same ad unit code for multiple ad slots.